Formula/radare2.rb

```diff
@@ -3,17 +3,17 @@ class Radare2 < Formula
   homepage "http://radare.org"
 
   stable do
-    url "http://radare.org/get/radare2-0.10.1.tar.xz"
-    sha256 "5ac02717786f2ff3b5326927351d5ca38464da89675c8edfb4ded43addb22987"
+    url "http://rada.re/get/radare2-0.10.2.tar.xz"
```

Comment:
The canonical links still appears to be
http://www.radare.org/get/radare2-0.10.2.tar.xz